solidity_contracts/src/EvmPrecompiles/EvmPrecompiles.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+/// @title EVM Precompiles Integration
+/// @notice Contract for integration testing of EVM precompiles.
+/// @dev Implements functions for ECADD and ECMUL precompiles.
+contract EvmPrecompiles {
+    /// @dev Address of the ECADD precompile
+    address private constant ECADD_PRECOMPILE = address(0x06);
+    /// @dev Address of the ECMUL precompile
+    address private constant ECMUL_PRECOMPILE = address(0x07);
+
+    /// @dev Gas cost for ECADD call is 150
+    uint256 private constant ECADD_GAS = 1000000;
+    /// @dev Gas cost for ECMUL call is 6000
+    uint256 private constant ECMUL_GAS = 1000000;
+
+    /*//////////////////////////////////////////////////////////////
+                               CONSTRUCTOR
+    //////////////////////////////////////////////////////////////*/
+    constructor() {}
```

Comment:
No need to explicitly write the default constructor. You can omit it as it is empty